src/components/KakaoMap/KakaoMapDetail.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Map, CustomOverlayMap } from 'react-kakao-maps-sdk';
+import styled from 'styled-components';
+
+const KakaoMapDetail = props => {
+  const { data, size } = props;
```

Comment:
```suggestion
const KakaoMapDetail = ( { data, size } ) => {
```
이렇게 구조분해 할당 하는 방법도 있습니다~

Reply:
피드백 반영하였습니다!